### PR TITLE
[WIP][9.0] Improve the calculation of current Payroll rules and fix travis

### PR DESCRIPTION
**Pull request by Wizr**
@victormartinelicocorp

### CODE REVIEW

**Review by martintorresvm — CHANGES_REQUESTED** [review]
@Wizr Create a separated PR just to add Travis File

**Comment by Wizr** [comment]
Travis file added #37 , but runbot test failed. This PR fix the problem, so they blocked each other. What to do next?

**Comment by martintorresvm** [comment]
@Wizr I think is normal

https://runbot2.odoo-community.org/runbot/static/build/3286675-37-61a868/logs/job_20_test_all.txt

We need to fix or know if the message `codecov/project — No report found to compare against`  is normal

So for Runbot #37 IMO is normal.
Waiting for OCA reviewers if they could help

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+language: python
```
Comment (elicoidal) [inline .travis.yml]: remove from this PR

**Review by elicoidal — CHANGES_REQUESTED** [review]
Please remove the travis file
Needs functional testing by Linda

**Review by martintorresvm — APPROVED** [review]

**Review by elicoidal — APPROVED** [review]

**Comment by elicoidal** [comment]
Waiting for functional tests to merge @Wizr @victormartinelicocorp

**Comment by Wizr** [comment]
Linda has tested, and found no issues.

**Pull request merged**